TS06-Facets-And-NDVI-Plotting-In-R.Rmd

```diff
@@ -79,39 +78,19 @@ recommend starting with the [Subset & Manipulate Time Series Data with `dplyr` l
 and the [Plotting Time Series with ggplot in R lesson]({{site.baseurl}}/R/Time-Series-Plot-ggplot/ "Learn ggplot")  
 respectively, to gain familiarity.
 
-###Time Series Lesson Series 
-This lesson is a part of a series of lessons on tabular time series data in `R`:
-
-* [ Brief Time Series in R - Simple Plots with qplot & as.Date Conversion]({{ site.baseurl}}/R/Brief-Tabular-Time-Series-qplot/)
-* [Understanding Time Series Metadata]({{ site.baseurl}}/R/Time-Series-Metadata/)
-* [Convert Date & Time Data from Character Class to Date-Time Class (POSIX) in R]({{ site.baseurl}}R/Time-Series-Convert-Date-Time-Class-POSIX/)
-* [Refining Time Series Data - NoData Values and Subsetting Data by Date in R]({{ site.baseurl}}/R/Subset-Data-and-No-Data-Values/)
-* [Subset and Manipulate Time Series Data with dplyr]({{ site.baseurl}}/R/Time-Series-Subset-dplyr/)
-* [Plotting Time Series with ggplot in R]({{ site.baseurl}}/R/Time-Series-Plot-ggplot/)
-* [Plot uisng Facets and Plot Time Sereis with NDVI data]({{ site.baseurl}}/R/Time-Series-Plot-Facets-NDVI/)
-* [Converting to Julian Day]({{ site.baseurl}}/R/julian-day-conversion/)
-
-###Sources of Additional Information
 
 </div>
 
-#The Data Approach
-What we want to do with ecological time series data often extends beyond
-plotting a single variable as it changes across time. For example, when studying
-phenology it is likely that a combination of precipitation, temperature, and
-solar radiation (PAR) plays a role in the annual greening and browning of 
-plants.  
-
-In this lesson we will learn how to plot several multiple variables variables
-together across the seasons, prior to adding in NDVI data.  Using the NDVI data 
-we can finally directly compare the observed plant phenology with patterns we've
-already been exploring.  
+#Plotting Subsetted Data Using Facets
+ 
+In this lesson we will learn how to create a panel of indivual plots - known as
+facets in `ggplot2`. Each plot represents a particular `data_frame` time-series 
+subset - for example year or season.
 
 ###Load the Data
-We will need the daily micro-meteorology data for 2009-2011 from the Harvard
-Forest. If you do not have these data sets as `R` objects, please load them from
-the .csv files in the downloaded data and convert date-time variables to 
-POSIXct date-time class. 
+We will use the daily micro-meteorology data for 2009-2011 from the Harvard
+Forest. If you do not have this data loaded into an `R` `data_frame`, please 
+load them, and convert date-time columns to a date-time class now.
```

Comment:
I realized there are two "data frames" after talking with a colleague.

`data.frame` = base R
`data_frame` = dplyr data frame

Let's check into it but it should become an issue and we should be consistent throughout in application.
Or we just use data frame everywhere and avoid the confusion. But i like to be specific if we can be.